int32 size
float64[] s_data
float64[] mfcc
float64 zero
float64 energy
float64 mfccl2
int32 vad_state
